src/MSBuild/CommandLineSwitches.cs

```diff
@@ -269,6 +270,7 @@ private struct ParameterizedSwitchInfo
             new ParameterizedSwitchInfo(  new string[] { "warnasmessage", "nowarn" },           ParameterizedSwitch.WarningsAsMessages,         null,                           true,           "MissingWarnAsMessageParameterError",  true,   false),
             new ParameterizedSwitchInfo(  new string[] { "binarylogger", "bl" },                ParameterizedSwitch.BinaryLogger,               null,                           false,          null,                                  true,   false),
             new ParameterizedSwitchInfo(  new string[] { "analyze", "al" },                     ParameterizedSwitch.Analyze,                    null,                           false,          null,                                  true,   false),
+            new ParameterizedSwitchInfo(  new string[] { "analyzeStats", "as" },                ParameterizedSwitch.AnalyzeStats,               null,                           false,          null,                                  true,   false),
```

Comment:
May I clarify why the switch is parameterized, what are the parameters of invocation?

Reply:
I changed it to a prameterless switch. Honestly did not pay a lot of attention to that when adding the command line switch.